### ISSUE TITLE
It is time to make the big decision on a fork for AMP takeover

### ISSUE THREAD

**Issue by Frenchomatic** [ahmedkaludi/accelerated-mobile-pages#2216]
Please read this post

https://wordpress.org/support/topic/would-like-to-understand/#post-10355868

I really think you are at the point of having to make a decision whether have two plugins.  You will avoid so many problems. At least discuss it. 

It is clear that those who wish to run in AMP takeover are in a different position to those that are trying to keep a mixed system. In the former, we are comfortable with using things like 

```
<amp-youtube width="1280"
  height="720"
  layout="responsive"
  data-videoid="xxxxx">
</amp-youtube>
```

or 

`<amp-frame`
.
.
.

We want the plugin to recognize this code and directly add this to the head for us. For example, <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>

What we have at present is one version recognizes it and then another doesn't and this leads to chaos and users having to continually update their web sites.

**Comment by ahmedkaludi** [comment]
Similar issue https://github.com/ahmedkaludi/accelerated-mobile-pages/issues/2215

**Comment by Frenchomatic** [comment]
Yes - I made this post having thought about it as opposed to a support question #2215 about the current version - one which meant I had to go through several sites where maps and videos were no longer showing - I had to make a decision whether to remove the AMP specific code and go back to regular iframes or keep the AMP specific code and recode the scripts in the head.

If you like this was the basic sequence of events: 
version 1: iframes worked
version 2: iframes no longer work so I use AMP specific code and embed the required scripts into the head. All is well and feeling pleased with myself.
version 3: Scripts now get embedded automatically into the head and therefore they are now appearing twice which produces validation errors. Remove the extra script in the head from a couple of dozen or so pages that I was placing them on. Al is well and it is working properly again. 
version 4: Scripts no longer get embedded automatically into the head - make a decision because iframes now work or put scripts back into the head myself across those pages. 

The basic problem is really about whether AMP takeover would be more stable if it were forked into another plugin.

**Comment by Sejiro-Hiko** [comment]
@Frenchomatic we'll be making the plugin to automatically detect the amp-components and add their respective scripts if they are not already there in the head.
